website/content/en/v0.26.1/upgrade-guide.md

```diff
@@ -41,6 +41,18 @@ Karpenter ships with a few Custom Resource Definitions (CRDs). These CRDs are pu
     ```
     helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version vx.y.z --namespace karpenter --create-namespace
     ```
+  * Upgrading from older karpenter version that did not include `awsnodetemplates.karpenter.k8s.aws` labels and annotations requires manual CRDs annotations before issuing `helm upgrade`. 
```

Comment:
Do you mind making this update in the `preview` along with the `v0.26.1`, `v0.26.0` and the `v0.27.0` docs. The `preview` docs are the ones that we carry forward each time we have a release.

Reply:
removed from 0.25 and made changes in `v0.26.1`, `v0.27.0` and `preview`. Removed changes from `v0.25.0`